packages/services/api/src/modules/auth/lib/supertokens-strategy.ts

```diff
@@ -184,144 +325,3 @@ const SuperTokenAccessTokenModel = zod.object({
   superTokensUserId: zod.string(),
   email: zod.string(),
 });
-
-function transformOrganizationMemberLegacyScopes(args: {
```

Comment:
These are now translated when reading from the database, as they are no longer used anywhere else.